armi/reactor/tests/test_blocks.py

```diff
@@ -1766,6 +1735,126 @@ def test_getReactionRates(self):
         )
 
 
+class BlockRotateTests(unittest.TestCase):
+    """Tests for the ability for a block to be rotated."""
+
+    BOUNDARY_PARAMS = [
+        "pointsCornerDpa",
+        "cornerFastFlux",
+        "pointsCornerFastFluxFr",
+        "pointsCornerDpaRate",
+        "pointsEdgeFastFluxFr",
+        "pointsEdgeDpa",
+        "pointsEdgeDpaRate",
```

Comment:
```suggestion
        "cornerFastFlux",
        "pointsCornerDpa",
        "pointsCornerDpaRate",
        "pointsCornerFastFluxFr",
        "pointsEdgeDpa",
        "pointsEdgeDpaRate",
        "pointsEdgeFastFluxFr",
```

Reply:
I just alphabetized because I couldn't tell if there were any duplicates, and I was too lazy to check the slow way.